ducktape/command_line/main.py

```diff
@@ -15,11 +15,11 @@
 from ducktape.tests.loader import TestLoader
 from ducktape.tests.runner import SerialTestRunner
 from ducktape.tests.reporter import SimpleStdoutReporter, SimpleFileReporter
-from ducktape.tests.session_context import TestSessionContext
+from ducktape.tests.session_context import SessionContext
 from ducktape.cluster.vagrant import VagrantCluster
 from ducktape.command_line.config import ConsoleConfig
 from ducktape.tests.session_context import generate_session_id, generate_results_dir
-from ducktape.unit_tests.mock import swap_in_mock_run, swap_in_mock_fixtures
+from tests.mock import swap_in_mock_run, swap_in_mock_fixtures
```

Comment:
Are these going to be permanent or are these mock things going away sometime soon? If they're permanent we should move them out of the tests/ module.